scripts/exrrfs_ioda_bufr.sh

```diff
@@ -35,9 +31,19 @@ for yaml in ${yaml_list[@]}; do
  ${MPI_RUN_CMD} ${execfile} ${yaml}
  # some data may not be available at all cycles, so we don't check whether bufr2ioda.x runs successfully
 done
-ls ./ioda*nc  
+
+# run offline IODA tools
+${cpreq} ${HOMErrfs}/sorc/RDASApp/rrfs-test/IODA/offline_add_var_to_ioda.py .
+ioda_files=$(ls ioda*nc)
+for ioda_file in ${ioda_files[@]}; do
+  python offline_add_var_to_ioda.py -o ${ioda_file}
```

Comment:
I think NCO wants the script (including *.py) to run from the command line directly without the leading "python".

Reply:
Also, has the Python environment been correctly loaded for this tool?

Reply:
@guoqing-noaa, thanks for the comments. The tool already had `#!/usr/bin/env python`  at the top. Its just habit to run it with `python` anyway. I just removed it in my most recent commit and it still works. As for the python environment it seems everything that is needed is already loaded so I didn't have to load anything else.